pywbem_mock/_wbemconnection_mock.py

```diff
@@ -1156,6 +1156,9 @@ def _imeth_GetInstance(self, **params):
         parameters defined for that method and map response to tuple response
         for imethodcall.
 
+        This method includes the namespace within the InstanceName
+        rather than as a separate element.
+
```

Comment:
The call to this method is done in `_mock_imethodcall()` which generically passes on its own arguments. So the `namespace` parameter will inevitably be passed to this method. 

Therefore, when this method determines the namespace to be used a few lines down from here, it should in addition take the `namespace` parameter into account.